db/data/20191029113201_backfill_note_on_already_checked_claims.rb

```diff
@@ -0,0 +1,22 @@
+require "csv"
```

Comment:
This doesn't appear to be using the `CSV` class. Can this be ditched?

Reply:
Oh, good call